Sources/main.cpp

```diff
@@ -319,24 +356,47 @@ static int GdbserverMain(int argc, char **argv) {
 
   bool reverse = opts.getBool("reverse-connect");
 
+#if defined(OS_POSIX)
```

Comment:
Move this block after creating a socket so you can minimize the number of ifdefs